### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by RitaDee**
### In this PR, I implemented the following:

- **Delete all the boilerplate** from Create React App (text, images, styles).
- **Create a new directory** called **components**.
- Inside **components**, create a new **Calculator.js file**.
- Implement the design only **(JSX and CSS)**
   - **Develop just the UI of the calculator**.
- Use **functional components**.
- Create at least **1 child component**.
- Once the Calculator is ready, import it and **use it in your top-level component: App.js**.
- **Test it in the browser**; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const Calculator = () => (
+  <div className="container">
+    <div className="result">0</div>
+    <div className="key">AC</div>
+    <div className="key">+/-</div>
+    <div className="key">%</div>
+    <div className="operator">/</div>
```
Comment (omar-muhamad) [inline src/components/Calculator.js]: - [ ] As per this [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Calculator.js%20file.-,In%20Calculator.js%2C%20create%20a%20React%20component%20that%20matches%20this%20design%3A,-Implement%20the%20design), Please replace the division operator with this symbol `÷` instead to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image |image |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @RitaDee,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used the correct workflow.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/Calculator.js (hunk 1 above)
Comment (RitaDee) [reply src/components/Calculator.js]: Noted with thanks @Omar-Muhamad.

**Review by victorokoroji — APPROVED** [review]
Hi @RitaDee, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**